support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -91,10 +90,8 @@ export const getThankYouModuleData = (
 				<FeedbackCTA
 					feedbackSurveyLink={
 						isOneOff
-							? 'https://guardiannewsandmedia.formstack.com/forms/guardian_supporter_single'
-							: amountIsAboveThreshold
-							? 'https://guardiannewsandmedia.formstack.com/forms/guardian_supporter_above'
-							: 'https://guardiannewsandmedia.formstack.com/forms/guardian_supporter_below'
+							? 'https://guardiannewsandmedia.formstack.com/forms/eoy_ny_23_24_onetime'
+							: 'https://guardiannewsandmedia.formstack.com/forms/eoy_ny_23_24_recurring'
```

Comment:
This could become: 

`https://guardiannewsandmedia.formstack.com/forms/eoy_ny_23_24_${isOneOff ? "onetime" :  "recurring"}`

Reply:
Yep, got it, good point, all done.